### PR TITLE
Fix animate and deal damage

### PR DESCRIPTION
**Pull request by juraj-visnovsky**

### CODE REVIEW

**Comment by martinkonopka** [comment]
Bot is now easy to beat :)

**Comment by martinkonopka** [comment]
After the merge I noticed that the opponent takes double damage.

File: src/server/game.js [hunk 1]
```diff
@@ -166,11 +166,11 @@ class Game {
     let spell = player.castSpell();
     this.animateEvent(player, "Attacking", spell);
 
-    let opponent = this.getOpponent(spell);
```
Comment (michal-granec) [inline src/server/game.js:169]: Toto bol taky fail :man_facepalming:

Comment (martinkonopka) [reply src/server/game.js:169]: Raz sa tento riadok vrati, ked bude mat spell vlastny target :)

**Pull request merged**